pypose/module/imu_preintegrator.py

```diff
@@ -66,11 +79,21 @@ def forward(self, dt, gyro, acc, rot:pp.SO3=None, gyro_cov=None, acc_cov=None, i
                 of the dictionary should be :obj:`{'pos': torch.Tensor, 'rot': pypose.SO3, 'vel':
                 torch.Tensor}`. The initial state given in constructor will be used if not given.
 
-        Note:
-            This layer supports the input shape with :math:`(B, F, H_{in})`, :math:`(F, H_{in})`
-            and :math:`(H_{in})`, where :math:`B` is the batch size (or the number of IMU),
-            :math:`F` is the number of frames (measurements), and :math:`H_{in}` is the raw
-            sensor inputs.
+        Shape:
+            - input (:obj:`dt`, :obj:`gyro`, :obj:`acc`): This layer supports the input shape with
+              :math:`(B, F, H_{in})`, :math:`(F, H_{in})` and :math:`(H_{in})`, where :math:`B` is
+              the batch size (or the number of IMU), :math:`F` is the number of frames (measurements),
+              and :math:`H_{in}` is the raw sensor signals.
+
+            - init_state (Optional): The initial state before the integration. It contains
+              :code:`pos`: initial position, :code:`rot`: initial rotation, :code:`vel`: initial
+              velocity, with the shape :math:`(B, H_{in})`
+
+            - output: The output is a :obj:`dict` of integration state.
+              It contains the ``pos``: position, ``rot``: rotation, ``vel``: velocity.
+              Each output has the shape :math:`(B, F, H_{out})`, where :math:`H_{out}` is
+              the corresponding signal dimension. If the flag :obj:`prop_cov` is ``True``, the
+              output will include ``cov``, the covariance matrix with shape :math:`(B, 9, 9)`.
 
         IMU Measurements Integration:
 
```

Comment:
what is `b_i^g` `b_i^a` and `b_i^a`? better to add an explanation.

Reply:
In our current implementation, `b_i^g` is not modeled in the `integrator`. The input to the integrator shall be "de-bias". I will suggest to remove this notation.

Reply:
I just changed my mind, maybe we will need to model the bias in our future. I will add the explanation.